sharder/sharded-bot.js

```diff
@@ -64,7 +67,9 @@ let token;
 
 try {
   password = require('discord-bot-maker');
-} catch {}
+} catch {
+  password = '';
+}
```

Comment:
password is already set as `''` so it s both redundant and pointless to do it again :)
```suggestion
} catch {}
```